apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -140,6 +141,28 @@ public string DataSource
             set => this[DataSourceKeyword] = ValidatedDataSource(value);
         }
 
+        /// <summary>
+        /// The universe domain to connect to, or null to use the default universe domain <see cref="DefaultDomain"/>.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// <see cref="UniverseDomain"/> is used to build the endpoint to connect to, unless <see cref="EndPoint"/>
+        /// is explicitly set (i.e. without using the default host and port), in which case <see cref="EndPoint"/> will be used without further modification.
```

Comment:
I think the part in parenthesis is confusing. And this whole thing should be:

"unless EndPoint or Host are explicitly set"